portal/templates/utilisation-report-service/utilisation-report-upload/utilisation-report-upload.njk

```diff
@@ -5,7 +5,7 @@
 {% set nextDueReport = dueReportDates[0] %}
 {% set genericWarningText = "There are overdue reports, please send them as soon as possible." %}
 {% set warningText %}
-    {{ genericWarningText if dueReportDates.length > 2 else [nextDueReport.reportPeriod, " report is overdue, please send it as soon as possible."] | join }}
+{{ genericWarningText if dueReportDates.length > 2 else [nextDueReport.reportPeriod, " report is overdue, please send it as soon as possible."] | join }}
 {% endset %}
```

Comment:
This indent looks intentional because it's within a `{% set` and `{% endset` block?

Reply:
The lint formats it like this and I wasn't sure why - I've re-added the indenting but may need to see about some specific nunjucks linting?

Reply:
Hm maybe...oh well!